src/common/intl.cpp

```diff
@@ -679,6 +679,8 @@ void wxLocale::AddLanguage(const wxLanguageInfo& info)
 /* static */
 const wxLanguageInfo* wxLocale::GetLanguageInfo(int lang)
 {
+    if (lang == wxLANGUAGE_DEFAULT)
+        lang = GetSystemLanguage();
```

Comment:
And the same thing here: we need to document that this is required, even if `wxUILocale` function already does the "same" thing internally because our `GetSystemLanguage()` is not the same.

In fact, maybe it would be better to use `wxUILocale::GetSystemLocale()` directly here, it could be more clear.

Reply:
> And the same thing here: we need to document that this is required, even if `wxUILocale` function already does the "same" thing internally because our `GetSystemLanguage()` is not the same.

Yes, for sure this should be documented.

> In fact, maybe it would be better to use `wxUILocale::GetSystemLocale()` directly here, it could be more clear.

I wanted to make clear that we use the "system language" as defined in the sense of `wxLocale`. If this _definition_ or its implementation would change, it could be easily forgotten to change both places - even if it is rather unlikely that the implementation of `wxLocale::GetSystemLanguage` changes once again.